ckanext/restricted/action.py

```diff
@@ -114,16 +114,37 @@ def _restricted_resource_list_accessible_by_user(context, resource_list, package
     restricted_resources_list = []
     user_name = logic.restricted_get_username_from_context(context)
     user_obj = context.get('auth_user_obj')
+    if user_obj:
+        user_obj.org_dict = _get_org_dict(user_name)
```

Comment:
convert to pass `user_org_dict` as `logic.restricted_check_user_resource_access` optional parameter